docs/features/optin.md

```diff
@@ -0,0 +1,106 @@
+# Opt in to Enable Experimental Direct Controllers
```

Comment:
Thank you so much for adding the doc, @acpana ! It's fairly comprehensive and actionable.

I think this (though a less technical version) should also be on our public doc if we plan to advertise this feature. @yuwenma do we plan to do that?